main.go

```diff
@@ -73,6 +80,12 @@ func main() {
 		glog.Fatal(err)
 	}
 
+	var metricsBundle *util.PrometheusMetrics
+	if *enableMetrics {
+		glog.Info("Enabling metrics")
```

Comment:
`enabling metrics collecting and exporting to Prometheus`.

Reply:
done